src/test/java/seedu/address/commons/util/EditUtilTest.java

```diff
@@ -0,0 +1,103 @@
+package seedu.address.commons.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.commons.util.EditUtil.EditPersonDescriptor;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_ADDRESS_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_BIRTHDAY_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_EMAIL_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_NAME_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_PHONE_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_FRIEND;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.person.Address;
+import seedu.address.model.person.Birthday;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Phone;
+import seedu.address.model.tag.Tag;
+
+
+
+class EditUtilTest {
+
+    @Test
+    public void copy_isEqual() {
+        EditPersonDescriptor base = new EditPersonDescriptor();
+        EditPersonDescriptor test = new EditPersonDescriptor(base);
+
+        assertEquals(base, test);
+    }
+
+    @Test
+    public void isAnyFieldEdited_true() {
+        EditPersonDescriptor test;
```

Comment:
Maybe a better name for this is editPersonDescriptor? test seems like a test case instead